web/src/components/molecules/Asset/AssetList/AssetListBody/index.tsx

```diff
@@ -0,0 +1,26 @@
+import styled from "@emotion/styled";
+
+import PageHeader, { Props as PageHeaderProps } from "@reearth-cms/components/atoms/PageHeader";
+import AssetListTable, {
+  AssetListTableProps,
+} from "@reearth-cms/components/molecules/Asset/AssetList/AssetListTable";
+
+type Props = {
+  pageHeader: PageHeaderProps;
+  tableProps: AssetListTableProps;
+};
+const AssetListBody: React.FC<Props> = ({ pageHeader, tableProps }) => {
```

Comment:
Sorry can you make this just AssetList/index.tsx?

Reply:
Done!